src/components/calcite-color-picker/calcite-color-picker.tsx

```diff
@@ -607,7 +596,13 @@ export class CalciteColorPicker {
   /** Sets focus on the component. */
   @Method()
   async setFocus(): Promise<void> {
-    focusElement(this.hexInputNode);
+    const { colorFieldScopeNode, el } = this;
+
+    if (colorFieldScopeNode) {
+      return focusElement(colorFieldScopeNode);
+    }
+
+    el.focus();
```

Comment:
Focusing on the host element here has the same effect as not focusing since it has the default tab-index.

I think this should be: 

```ts
async setFocus(): Promise<void> {
  return focusElement(this.colorFieldScopeNode);
}
```

Reply:
fixed